src/app/characters/characters.service.spec.ts

```diff
@@ -4,16 +4,24 @@ import { TestBed } from '@angular/core/testing';
 import { environment } from '../../environments/environment';
 import { Character } from '../core/character.model';
 import { ContextService } from '../core/core.service';
+import { MarvelResponseData } from '../core/marvel-response.model';
 import { CharacterService } from './characters.service';
 
-const fakeCharacters = [{ name: 'Fake character' }] as Character[];
+const fakeMarvelResponseData = {
+  results: [{ name: 'Fake character' }] as Character[],
+  total: 1
+} as MarvelResponseData<Character>;
```

Comment:
We could define the type directly as `MarvelResponseData<Character>` instead of casting it. Is that so?

Reply:
Done.